docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -918,4 +918,202 @@ Amplify Ally:
     PP: 0
     Description: Move the chess pieces on your board.
     Mechanic: Select one ally. They can immediately move up to 4 spaces and take one Action.  
-    
\ No newline at end of file
+
+Craft Light:
+    Category:
+        - Magic
+    Description: Your skills let you conjure illusory effects.
+    Mechanic: Make a DR 3 Craft Check. On a Success you create an 8 space radiance of light for 15 minutes. On a Failure, the light only lasts for 1 round. You can create a temporary illusory object or infuse an existing object with this property. It can be covered and dimmed. Using a Minor action allows you to extinguish a light. If the light is more than 200 spaces away from you, it disappears
+    Type: Minor
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: 
+            - Caster
+            - Support
+        Skill: Craft > 0
+
+Hidden Strike: 
+    Type: Passive
+    Category: 
+        - Stealth
+        - Combat
+        - Assassin    
+    Mechanic: When attacking an enemy that is unaware of your presence or engaged with an ally within 1 space, you draw with Upper Hand. Any attacks that are successful deal +1 damage
+    Description: You move silently in the shadows, and are especially deadly when unseen. 
+    XP: 2
+    Prereq:
+        Role: Martial
+        Level: 2
+        Skill: Stealth > 0
+
+Vanishing Step: 
+    Type: Major
+    Category: 
+        - Movement
+        - Stealth
+        - Assassin
+    Mechanic: Choose a point you can see within 10 spaces of you. If the space is unoccupied, you appear there Hidden. If you attempt to occupy a space with another creature in it you are not Hidden
+    Description: Sometimes, moving in the shadows is easiest when no one knows where you're going.
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Martial
+        Level: 3
+        Skill: Stealth > 1
+        Power: Hidden Strike
+    Range: 10
+    Target: 1
+    Save:
+        Trigger: If the space is occupied, the creature occupying that space makes a 
+        Type: STR
+        Fail: they are pushed out of the way
+        Succeed: instead, choose a space within 1 space of the creature to appear. 
+
+Summon Creature:
+    Type: Major
+    Category:
+        - Companion
+        - Summoner
+    Mechanic: Summon a creature to assist you. This creature uses the 'Summoned Creature' stat block and is treated as a Companion for the duration. You choose the creature's form. 
+    Description: It's good to have a friend, even when you have to summon them from somewhere else. Who ever said no to a little extra help?
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Caster
+        Level: 2
+        Skill: Knowledge > 0
+    Duration: 1 hour
+
+Phase Swap:
+    Type: Minor
+    Category:
+        - Companion
```

Comment:
Currently, categories are treated as embedded sections. The current set of powers, for example, would produce the following table of contents
- Combat
   - Control
   - Mystic Attacks
   - Support
       - Distract
   - Weapon Attacks

Here, is summoner a subset of movement? If these are independent items, use the 'Tags' key

Reply:
Got it, that makes sense. Updated the Powers YAML to reflect this by adding 'Tags' key to represent each Archetype named Skill Tree and kept the hierarchy clear.